### PR TITLE
ref: Tweak rrweb configuration

### PR DESCRIPTION
**Pull request by dcramer**
- Reduce snapshot to 1 minute (should reduce total filesize)
- Update to 0.1.1 which supports upstream maskAllInputs configuration

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -13785,13 +13790,14 @@ rrweb-snapshot@^0.7.26:
   resolved "https://registry.yarnpkg.com/rrweb-snapshot/-/rrweb-snapshot-0.7.26.tgz#339769cdcfc53db4fec279501706ff6129671cb2"
   integrity sha512-z4k6swLNI5YPDFFcYp19Ggo2kE12cWG1OZCanlaMvwSOi82OOZ229ckYBCPYHbYQnCwuBF5KSMZi2i0CwpWHng==
 
-rrweb@^0.7.25, rrweb@^0.7.30:
-  version "0.7.30"
-  resolved "https://registry.yarnpkg.com/rrweb/-/rrweb-0.7.30.tgz#5a2ba3e667b5f663fc207574188fcfcd450467df"
-  integrity sha512-ODA0W2cas5yoU1jLO/UVNUrr99jIfv5nQ7jrD7gfFG09O0YaMV9I9p5SkAztQe6gv2QvxgFsyFxzWjpfD7btvw==
+rrweb@^0.7.25, rrweb@^0.7.33:
```
Comment (dcramer) [inline yarn.lock:13793]: hrm is this maybe because of rrweb-player or something? whys it end up w/ two versions in here

Comment (billyvg) [reply yarn.lock:13793]: ya `rrweb-player`

**Review by billyvg — APPROVED** [review]

**Comment by dcramer** [comment]
will dig into rrweb-player to see if there's a more clear way to version to avoid two rrweb versions

**Comment by billyvg** [comment]
@dcramer It gets resolved to use `.33` I'm p sure

**Comment by dcramer** [comment]
looks like rrweb-player still pinned to a lesser version. i may push a fix upstream so it doesn't pin there

**Pull request merged**